### PR TITLE
First draft Terms of Service

### PR DESCRIPTION
**Pull request by sebrose**
Linked to by proposed EULA, so needs to promoted first.

Is this too wordy, or can we live with it?

### CODE REVIEW

**Comment by mattwynne** [comment]
LGTM but I'm a bit confused - this is going up on the cucumber.io site and seems to refer to the website only - not Pro. I'd assume we'll need different and more specific terms for use of pro, won't we? Or is that in the EULA?

**Comment by sebrose** [comment]
That's in the EULA

On Tue, 5 Jul 2016, at 12:18 PM, Matt Wynne wrote:

> LGTM but I'm a bit confused - this is going up on the cucumber.io site
> and seems to refer to the website only - not Pro. I'd assume we'll
> need different and more specific terms for use of pro, won't we? Or is
> that in the EULA?
> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[1], or mute the thread[2].

## 

The Cucumber for Java Book - Seb Rose, Matt Wynne & Aslak Hellesøy
Now available from the Pragmatic Press -
https://pragprog.com/book/srjcuc/the-cucumber-for-java-book

Links:
1. https://github.com/cucumber/website/pull/157#issuecomment-230451901
2. https://github.com/notifications/unsubscribe/ACGh7jbYtXLxgR5kkG6JD9fZSKnd-81rks5qSj1_gaJpZM4JE8Vm
